url.bs

```diff
@@ -284,9 +284,13 @@ U+005C (\), or U+005D (]).
 <ol>
  <li><p>If <var>host</var> is not a <a>domain</a>, then return null.
 
- <li><p>Return the <a for=host>public suffix</a> obtained by executing the
- <a href="https://publicsuffix.org/list/">algorithm</a> defined by the Public Suffix List on
- <var>host</var>. [[!PSL]].
+ <li><p>Let <var>publicSuffix</var> be the public suffix determined by running the
+ <a href="https://publicsuffix.org/list/">Public Suffix List algorithm</a> with <var>host</var> as
+ domain. [[!PSL]]
+
+ <li><p>Assert: <var>publicSuffix</var> is encoded using A-Labels. [[!UTS46]]
```

Comment:
UTS 46 doesn't really define what an A-label is, as it only seeks to define a replacement for the processing algorithm. In fact, it defers the definition to [RFC 5890](https://tools.ietf.org/html/rfc5890) and doesn't use the word at all itself.

Additionally, the term "A-label" exclusively refers to validly encoded labels with `xn--` prefix. To refer to all ASCII-encoded labels, it may be better to use either of the following:

> Assert: _publicSuffix_ contains only ASCII characters.

or

> Assert: Labels in _publicSuffix_ must be either A-labels or NR-LDH labels. [RFC5890]

Reply:
I wanted to use UTS46 as an intermediary, but as A-Label doesn't work and NR-LDH isn't mentioned there we'll just assert ASCII code points.